### PR TITLE
[Merged by Bors] - feat: Port/Topology.UniformSpace.Pi

### PR DESCRIPTION
**Pull request by arienmalec**
port of topology.uniform_space.pi

all simple fixes
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/UniformSpace/Pi.lean [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+section
+
+open Filter UniformSpace
+
+universe u
+
+variable {ι : Type _} (α : ι → Type u) [U : ∀ i, UniformSpace (α i)]
+
+-- porting note: no include
+--include U
```
Comment (dupuisf) [inline Mathlib/Topology/UniformSpace/Pi.lean:32]:
```suggestion
```
This can just be deleted safely, no reason to flag it.

**Review by dupuisf — COMMENTED** [review]
Thanks!

bors d+

**Comment by arienmalec** [comment]
r+

**Pull request closed without merging**